docs/extend/modular-resources/_index.md

```diff
@@ -10,679 +10,26 @@ aliases:
     - "/program/extend/modular-resources/"
 ---
 
-The Viam module system allows you to integrate custom {{< glossary_tooltip term_id="resource" text="resources" >}}, ([components](/components/), and [services](/services/)) into any robot running on Viam.
+The Viam module system allows you to integrate custom {{< glossary_tooltip term_id="resource" text="resources" >}} into any robot running on Viam.
+The {{< glossary_tooltip term_id="rdk" text="RDK">}} [manages](/extend/modular-resources/key-concepts/) these modular resources like the resource types and models that are built-in to the kit.
```

Comment:
```suggestion
The {{< glossary_tooltip term_id="rdk" text="RDK">}} [manages](/extend/modular-resources/key-concepts/) modular resources and built-in resources in the same way.
```
or can we avoid talking about RDK?
```suggestion
`viam-server` [manages](/extend/modular-resources/key-concepts/) modular resources and built-in resources in the same way.
```
(add link on viam-server maybe--and maybe we should add viam-server to the glossary but maybe that's for a different ticket)

Reply:
the latter I think